01-faq-accordion-card/code/src/components/Accordion.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Data } from "./AccordionData";
+import picture3 from "../assets/icon-arrow-down.svg";
+const AccordionSection = styled.div`
```

Comment:
Move styled components to separate file
https://github.com/OleksiiSamutin/DSC_JAVAscript_StudyGroup/pull/1#discussion_r606842188